docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -29,6 +29,9 @@ Unexpected behavior may occur if multi-tenancy is only enabled in some component
 :::caution
 
 Multi-tenancy is currently only available for Camunda 8 Self-Managed with authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/).
+For a standalone Zeebe instance, multi-tenancy is currently available with authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/)
+or by [integrating an external tenant-managing component](../zeebe-deployment/zeebe-gateway/interceptors.md#implementing-a-tenant-providing-interceptor)
+with Zeebe.
```

Comment:
:question: @christinaausley I'm interested about your opinon on mentioning this feature in the conceptual multi-tenancy docs, as this document is mainly about Camunda 8, and not really about a standalone Zeebe instance. 

However, a standalone Zeebe instance can still be considered a self-managed setup.

Reply:
I'm fine with this 👍